docker-compose-dev.yml

```diff
@@ -173,7 +173,7 @@ services:
 
   elasticsearch:
     restart: "unless-stopped"
-    image: elasticsearch:5
+    image: elasticsearch:7.10.1
```

Comment:
Shall we just use 7.10 in order to have latest bugfix updates? (Or even 7?) The docker compose versions used only for development, the production upgrades are handled differently.

Reply:
I used `elasticsearch:7` but I got this error: `ERROR: manifest for elasticsearch:7 not found: manifest unknown: manifest unknown`.

when I looked into `elasticsearch` docker hub, there are two tags that are available: `7.10.1` and `6.8.13`. I thought of putting `latest` but this tag is not supported.

https://hub.docker.com/_/elasticsearch

Reply:
Perhaps we should be using the same ES as Invenio, see [here](https://github.com/inveniosoftware/cookiecutter-invenio-instance/blob/55645e9b56db0f52ae0235efa68c72bf173c1d40/%7B%7Bcookiecutter.project_shortname%7D%7D/docker-services.yml#L78), it seems to be the [recommended way](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html).

Reply:
Using the same as `invenio` sounds good. It is `7.9.3` which was released 3 months ago and `7.10.1` was released one month ago.